recipes/trilium/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "trilium",
+  "name": "Trilium",
+  "version": "1.0.0",
+  "license": "MIT",
+  "config": {
+    "hasCustomUrl": true
```

Comment:
Do I understand correctly that the webapp of Trilium is necessarily self-hosted?

Reply:
yes